src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/Form.js

```diff
@@ -67,6 +67,7 @@ class Form extends React.Component<Props> {
                 options: {
                     idQueryParameter,
                     resourceKey,
+                    apiOptions,
                     routerAttributesToFormStore,
```

Comment:
might be more consistent to rename this option to something like `routerAttributesToApiOptions`?

Reply:
I like the `ToFormStore` suffix, because there is also `routerAttributesToEditRoute`, so the naming is consistent there. Question is if we maybe shouldn't try to have the same naming as in the Datagrid view, and adapt the term `apiOptions` to better match the existing options? But I think it would also be fine for me the way it is now.